autorest/codegen/templates/operation_tools.jinja2

```diff
@@ -1,3 +1,23 @@
+{% import 'keywords.jinja2' as keywords with context %}
+{# get method signature #}
+{% macro method_signature(operation, operation_name, async_mode, is_paging) %}
```

Comment:
I'm concerned to see "is_paging", I don't like to make op_tools specific of a given feature name. I'd prefer something generic like "coroutine=True/False" to tell if we need the keyword "async" or not